vms/platformvm/vm.go

```diff
@@ -274,6 +281,42 @@ func (vm *VM) Initialize(
 	return nil
 }
 
+func (vm *VM) startMempoolPruner() {
+	ticker := time.NewTicker(mempoolPrunerFrequency)
+	defer ticker.Stop()
+
+	for {
+		select {
+		case <-vm.onShutdownCtx.Done():
+			return
+		case <-ticker.C:
+			err := vm.PruneMempool()
+			vm.ctx.Log.Debug("pruning mempool failed",
```

Comment:
Seems like this log should either not say that it failed or that it should be conditional on a non-nil error